src/subscriptions/TyranoDiagnostic.ts

```diff
@@ -196,38 +286,65 @@ export class TyranoDiagnostic {
 
             if (this.isValueIsIncludeVariable(data["pm"]["target"])) {
               if (!this.isExistAmpersandAtBeginning(data["pm"]["target"])) {
-                let diag = new vscode.Diagnostic(range, "パラメータに変数を使う場合は先頭に'&'が必要です。", vscode.DiagnosticSeverity.Error);
+                const diag = new vscode.Diagnostic(
+                  range,
+                  "パラメータに変数を使う場合は先頭に'&'が必要です。",
+                  vscode.DiagnosticSeverity.Error,
+                );
                 diagnostics.push(diag);
                 continue;
               }
-            } else if (!this.isValueIsIncludeVariable(data["pm"]["storage"])) {//targetがundefinedじゃない &&storageがundefinedじゃない && storageが変数でもない
+            } else if (!this.isValueIsIncludeVariable(data["pm"]["storage"])) {
+              //targetがundefinedじゃない &&storageがundefinedじゃない && storageが変数でもない
               //targetから*を外して表記ゆれ防ぐ
```

Comment:
```suggestion

              //targetから*を外して表記ゆれ防ぐ
```

直前の行のコメントと混ざらないように改行したほうがいいかもしれません